app/views/contestants/index.html.erb

```diff
@@ -0,0 +1,6 @@
+<% @contestants.each do |contestant| %>
+  <h3><%= contestant.name %></h3>
+  <p>Projects: <% contestant.projects.each do |project| %>
+    <%= project.name %> </p>
+  <% end %>
```

Comment:
Technically this works, but the user story asked to see the Projects listed on one line comma separated.